rootfs/ingest/python/qserv/ingest.py

```diff
@@ -128,10 +124,16 @@ def database_register_and_config(
         felis: `dict`, optional
             Felis schema for tables. Defaults to None.
         """
-        self.repl_client.database_register(self.contrib_meta.json_db)
-        self.repl_client.database_register_tables(self.contrib_meta.get_ordered_tables_json(), felis)
+        self.repl_client.database_register(self.contrib_meta._json_db)
```

Comment:
It looks like you're accessing a private member `_json_obj`? Why was this change from the public to the private access?

Reply:
:+1: